transactions/transactions_service.go

```diff
@@ -795,12 +794,17 @@ func (svc *transactionsService) validateCanPay(tx *gorm.DB, appId *uint, amount
 			balance := queries.GetIsolatedBalance(tx, appPermission.AppId)
 
 			if amountWithFeeReserve > balance {
+				message := NewInsufficientBalanceError().Error()
+				if description != "" {
+					message += " Invoice description: " + description
```

Comment:
do we need that "invoice description" addition?

Reply:
I guess not. I have removed it, thanks